dbms/src/Flash/Coprocessor/StreamingDAGResponseWriter.cpp

```diff
@@ -157,7 +162,7 @@ void StreamingDAGResponseWriter<StreamWriterPtr, enable_fine_grained_shuffle>::e
             for (const auto & block : input_blocks)
             {
                 chunk_codec_stream->encode(block, 0, block.rows());
-                packet.add_chunks(chunk_codec_stream->getString());
+                packet.add_chunks(chunk_codec_stream->getStringAndTrackMem(tmt));
```

Comment:
The `string` is moved to `packet`, will this make the memory tracked twice?

Reply:
actually no, since there is no branch, NRVO optimization tech will be applied